src/content/fr/2019/markup.md

```diff
@@ -0,0 +1,290 @@
+---
+part_number: I
+chapter_number: '3'
+title: Markup
+description: Chapitre de balisage d'Almanac Web 2019 couvrant les éléments utilisés,
+  éléments personnalisés, valeur, produits et en cas d'utilisation commune.
+authors:
+- bkardell
+reviewers:
+- zcorpan
+- tomhodgins
+- matthewp
+discuss: '1758'
+published: '2019-11-11 00:00:00 +0000'
+last_updated: '2019-11-23 00:00:00 +0000'
+---
```

Comment:
Can you review this agains the original please?

Issues noticed:
- some parts have got quotes (e.g. `chapter_number: '3'` instead of `chapter_number: 3`).
- some newlines added  (e.g. in description, authors, reviewers...etc.)
- some `[` and `]` has changed (e.g. in authors, reviewers).
- Timestamps seem to have changed format
- let's set the published date to the date you submit the PR (2019-11-27T00:00:00.000Z) and set last_updated to the same. We'll probably do a bulk update when we actually publish this online but for now let's just go with the date you submit the PR.

Also please add the following new line to this (after the reviewers line):
`translators: [SilentJMA]`
As per #507.

Can you also add yourself to the `src/config/2019.json` file as part of this PR with the translators role?